moveit_ros/planning/planning_request_adapter_plugins/src/check_start_state_bounds.cpp

```diff
@@ -115,8 +115,9 @@ class CheckStartStateBounds : public planning_interface::PlanningRequestAdapter
             {
               changed_req = true;
             }
+            break;
           }
-          break;
+          [[fallthrough]];  // If the revolute joint is non-continuous, we want to enter default case.
```

Comment:
don't we need to re-order this so that case REVOLUTE is immediately before the 'default'?

Reply:
Yeah, I think so.

For some reason I thought C++ kept checking the other cases, and not execute them all.